apps/modernization-ui/src/components/button/Button.module.scss

```diff
@@ -17,3 +17,11 @@
     line-height: 162%;
     font-style: normal;
 }
+
+.unstyled {
+    text-decoration: none !important;
+
+    &:hover {
+        text-decoration: underline !important;
+    }
```

Comment:
issue: Unstyled button is not present in our design system so we default to [USWDS](https://designsystem.digital.gov/components/button/) which has it styled like a link.